guides/NEW_APP.md

```diff
@@ -10,32 +10,28 @@ Prerequisites
 
 To follow this guide you will need:
 
-* A Github account with access to the ONSdigital organisation
+* A GitHub account with access to the ONSdigital organisation
 
-* A login for the concourse CI environment
+* A login for the Concourse CI environment
 
 * [dp-cli](https://github.com/ONSdigital/dp-cli)
 
 * Git:
 
   `$ brew install git`
 
-* Terraform, in document `dp-setup/terraform/README.md` install the version specified in the Prerequisites section : [dp-setup](https://github.com/ONSdigital/dp-setup/blob/develop/terraform/README.md#prerequisites) 
+* Terraform: follow the instructions in the [Terraform's Prerequisites section of dp-setup](https://github.com/ONSdigital/dp-setup/blob/awsb/terraform/README.md#prerequisites) to install the specified version. Do not install Terraform directly from Homebrew, as it will install the latest version by default
 
-  If for example that specifies version 0.13, then do:
-  
-  `$ brew install terraform@0.13`
-
-* Fly CLI (can be downloaded from the links in the corner of the concourse UI)
+* Fly CLI (can be downloaded from the links in the corner of the Concourse UI)
 
 Getting started
 ---------------
 
-These steps will create a new app that is built by CI and deployed to the develop and production environments.  If any of the following Github links 404, likely you need to be added to the ONSdigital organisation.
+These steps will create a new app that is built by CI and deployed to the develop and production environments. If any of the following GitHub links 404, likely you need to be added to the ONSdigital organisation.
 
 1. Determine what language and technologies you will use
-1. Pick a local dev port for your app and add it to the [local ports list](PORTS.md)
-1. Use [`dp-cli` to generate the new repo with boilerplate files](https://github.com/ONSdigital/dp-cli/tree/main/project_generation/COMPLETE_PROJECT_SETUP.md)
-1. [Add the new app to the environment](https://github.com/ONSdigital/dp-setup#adding-a-new-app)
-1. [Add configurations for your new app](https://github.com/ONSdigital/dp-configs#adding-a-new-app) *Note: The Concourse CI pipeline `pipeline-generator` will auto-generate the pipeline for the new app.*
-2. Check the [Concourse UI](https://github.com/ONSdigital/dp/blob/main/training/platform-services/PLATFORM.md#concourse) and/or [Nomad UI](https://github.com/ONSdigital/dp/blob/main/training/platform-services/PLATFORM.md#nomad) to ensure your new app has successfully deployed to develop. See [here](https://github.com/ONSdigital/dp-setup/tree/develop/scripts#ansible-vault-helpers) for info on getting any acl tokens.
+2. Pick a local dev port for your app and add it to the [local ports list](PORTS.md)
```

Comment:
I think this renumbering is wrong, I think it's [recommended](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists) to use `1.` (sic) for all items in a numbered list, so only that `2.` (old line 41, new line 37) should be changed to `1.`  for this renumbering.

Reply:
I think it may be a matter of preference, since my IDE (JetBrains) complains if I set them all to "1". However, setting them all to 1 does have an advantage: when you need to add a new step in between you don't need to alter all the lines afterwards to update their number, if that makes sense. In any case, I've changed it as you suggested 👍